### DISCUSSION TITLE
Use bun for frontend development

### DISCUSSION THREAD

**Discussion by lafriks** [woodpecker-ci/woodpecker#2769] [General]
I have not checked if everything works yet but at least the dependency install works much faster as well as build. Wanted to hear opinions before I experiment further

**Comment by anbraten** [comment]
Never used it in production before, but sounds pretty cool. In the end its a question like always of does it help more than adding maintenance overhead or frustration if its not working.

**Reply by lafriks** [reply]
In theory it should be better especially as we are using typescript as it has built in typescript transpiler but need to test if everything works as expected

**Comment by pat-s** [comment]
Sounds interesting! Leaving this here for reference: https://bun.sh/blog/bun-v1.0
